plugins/aws/wafv2/wafv2WebAclLoggingEnabled.js

```diff
@@ -0,0 +1,65 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'AWS WAFV2 Web ACL Logging Enabled',
+    category: 'WAF',
+    domain: 'Availability',
+    severity: 'Medium',
+    description: 'Ensure that your Amazon WAF Web Access Control Lists are configured to capture information about all incoming requests.',
+    more_info: 'Enabling logging for Web ACL allows detailed logging of web requests that match defined rules within the WAF Web ACL. This enables comprehensive monitoring, analysis, and troubleshooting of security threats and traffic patterns within your web application',
+    link: 'https://docs.aws.amazon.com/waf/latest/developerguide/logging-management.html',
+    recommended_action: 'Modify WAF Web ACL and enable logging',
+    apis: ['WAFV2:listWebACLs','WAFV2:getLoggingConfiguration'],
+    realtime_triggers: ['wafv2:CreateWebACL', 'wafv2:DeleteWebAcl', 'wafv2:PutLoggingConfiguration'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = { };
+        var regions = helpers.regions(settings);
+
+        async.each(regions.wafregional, function(region, rcb){
+            var listWebACLs = helpers.addSource(cache, source,
+                ['wafv2', 'listWebACLs', region]);
+
+            if (!listWebACLs) return rcb();
+
+            if (listWebACLs.err || !listWebACLs.data) {
+                helpers.addResult(results, 3,
+                    'Unable to list WAFV2 web ACLs: ' + helpers.addError(listWebACLs), region);
+                return rcb();
+            }
+            
+            for (let webAcl of listWebACLs.data){
+                if (!webAcl.ARN) continue;
+
+                let getLoggingConfiguration = helpers.addSource(cache, source,
+                    ['wafv2', 'getLoggingConfiguration', region, webAcl.ARN]);
+
+                if (getLoggingConfiguration.err && 
```

Comment:
i think we should first check for the unknown case and handle the rest of the cases in the else part